attributes/default.rb

```diff
@@ -1,14 +1,15 @@
 include_attribute "kagent"
 
-default['ndb']['majorVersion']                        = "21"
-default['ndb']['minorVersion']                        = "04"
-default['ndb']['patchVersion']                        = "15"
+default['ndb']['majorVersion']                        = "22"
```

Comment:
version change should wait for RONDB-519, which updates rondb from the entire platform

Reply:
ok